### PR TITLE
First run at updating documentation to use DCOO instead of CLA

### PR DESCRIPTION
**Pull request by minddrive**
Updated dates and initial attempt to replace the CLA (Contributor License Agreement) with the DCOO (Developer's Certificate Of Origin).  Still needs work.

### CODE REVIEW

File: Legal.md [hunk 1]
```diff
@@ -56,13 +56,15 @@ There are only three steps needed to apply copyright and license properly in you
...
 
 For instance, what if if(we) actually _were_ to create a non-profit foundation for supporting and administering all of its open source projects? In order to reassign these projects to the purview of the foundation, if(we) would need to receive approval from each and every copyright holder for each and every project. As each contributor holds copyright over their own work, this could mean dozens or even hundreds of copyright holders…for **each** project. Considering how often people move around, change email addresses, or otherwise go AWOL, tracking them all down in order to get their approval to transfer the project to the foundation would be a logistical and administrative nightmare.
 
-Enter the [Contributor License Agreement](./starter_box/CLA.md) (aka the CLA).
+Enter the [Developer's Certificate Of Origin](./starter_box/DCOO.md) (aka the DCOO).
+
+NOTE: ALL the below about the CLA needs to be replaced with information on the DCOO
```
Comment (bugfood) [inline Legal.md]: The basic idea is that:
1. We don't want people to contribute code to us that is not their own and which is already under some other copyright.
2. The DCO asks contributors to certify that they are actually the authors and/or allowed to contribute the code to the project.

The specifics are covered within the DCO itself.

File: Legal.md [hunk 2]
```diff
@@ -64,25 +64,18 @@ For instance, what if if(we) actually _were_ to create a non-profit foundation f
 
 Enter the [Developer's Certificate Of Origin](./starter_box/DCOO.md) (aka the DCOO).
 
-NOTE: ALL the below about the CLA needs to be replaced with information on the DCOO
-
-A CLA is just a specialized form of license. Just as the Apache License grants the right to use copyrighted software, a CLA is a license which grants the right for an individual or organization to use the work of the contributor. The contributor retains copyright over the work, but if(we) is allowed to distribute and administer the work as needed.
+The DCOO ensures that any contribution made to the project was created by the contributor and that it is not under any other copyright.  It also ensures that the contributor must certify that they are allowed to contribute the code and is under the appropriate open source license.
```
Comment (bugfood) [inline Legal.md]: Grammar nit--I think this is better:
"contribute the code and that the code is under"

Comment (minddrive) [inline Legal.md]: Updated.

File: Legal.md [hunk 3]
```diff
@@ -64,25 +64,18 @@ For instance, what if if(we) actually _were_ to create a non-profit foundation f
...
+> Because of the potential legal and administrative snags involved, it is **absolutely vital** that there is a pre-receive hook to ensure the Signed-Off-By is in place for any external contribution into the project.
 
-The process itself is quite simple:
+The process for the above is straightforward:
 
-1. Contributor sends in a pull request.
-1. You ask the contributor to send in a [signed CLA](./starter_box/CLA_Process.md).
-1. if(we) Legal files away the signed CLA then lets you know it's been received.
-1. You merge the pull request (after reviewing and approving it, naturally).
+1. Contributor sends in a pull request, with the commit(s) using a '--signoff' or '-s'.
```
Comment (bugfood) [inline Legal.md]: I think I would split this up:
1. Contributor creates commits using 'git commit --signoff' (or '-s' for short).
2. Contributor sends in a pull request.

**Comment by bugfood** [comment]
Seems good to me other than squashing and the minor comments I left.

**Pull request merged**